recommendation/microsoft.owin.json

```diff
@@ -139,13 +139,26 @@
           "Description": "Add a comment to explain how to remove the base class of OwinMiddleware.",
           "Actions": [
             {
-              "Name": "AddComment",
+              "Name": "RemoveBaseClass",
               "Type": "Class",
-              "Value": "Please remove the base class here : OwinMiddleware",
-              "Description": "Add a comment to remove the base class OwinMiddleware",
+              "Value": "OwinMiddleware",
+              "Description": "Remove the base class OwinMiddleware",
               "ActionValidation": {
-                "Contains": "Pleaseremovethebaseclasshere:OwinMiddleware",
-                "NotContains": ""
+                "Contains": "",
+                "NotContains": ":OwinMiddleware"
+              }
+            },
+            {
+              "Name": "ReplaceMethodModifiers",
+              "Type": "Class",
+              "Value": {
+                "methodName": "Invoke",
+                "modifiers": "public"
+              },
+              "Description": "Remove the override modifier from the invoke method in owin middleware.",
+              "ActionValidation": {
+                "Contains": "public",
+                "NotContains": "override"
               }
```

Comment:
Per our discussion: would it make sense to rename this to `SetMethodModifiers` since we're reassigning the modifier list to the value specified in "modifiers" rather than doing a specific replacement? Such a change may create some pattern conflicts if other actions are following the same naming/logic convention so we should take that into consideration as well.

Reply:
Good idea, I went ahead and changed the descriptions for now so as to be much more precise and provide more details to remove all confusion.